### PR TITLE
OCM-238: Supporting permission boundary ARN

### PR DESCRIPTION
**Pull request by bardielle**

### CODE REVIEW

**Comment by sagidayan** [comment]
> Why not `permissions_boundary_arn` so it is clearer?

Probably more relevant in the provider itself... then here

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ This terraform module tries to replicate rosa CLI roles creation so that:
...
 |------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 |cluster_id| string      | Cluster ID                                                                                                                                         | "11111111111111111111111111111111"                                                                                                                                                         |
+|permissions_boundary| string      | The ARN of the policy that is used to set the permissions boundary for the IAM roles in STS clusters. | "arn:aws:iam::123456789012:policy/XCompanyBoundaries"                                                                                                                                                                        |
```
Comment (oriAdler) [inline README.md:31]: Maybe role boundaries?

**Review by oriAdler — COMMENTED** [review]
Left some small comments, other than that, LGTM from ROSA's perspective.

**Comment by bardielle** [comment]
@sagidayan -  I chose that attribute name because this is the attribute name in ROSA CLI (and AWS)

File: README.md (hunk 1 above)
Comment (bardielle) [reply README.md:31]: I copied it from ROSA CLI.... I can change it to be :
The ARN of the policy that is used to set the role boundaries for the IAM roles in STS clusters.

What do you think?

Comment (oriAdler) [reply README.md:31]: Sorry, I meant for the example:
`"arn:aws:iam::123456789012:policy/RoleBoundaries"`

**Review by sagidayan — APPROVED** [review]

**Pull request merged**